pystats_docs.md

```diff
@@ -0,0 +1,81 @@
+# pystats documentation
+
+This describes the meaning of the various sections and fields in the pystats output.
+
+## Execution counts
+
+This is the count of how many times each Tier 1 instruction is executed.
+
+The "miss ratio" column shows the percentage of times the instruction executed that it deoptimized.
+
+## Pair counts
+
+Shows the Top 100 most frequently occurring pairs of instructions as executed.
+
+## Predecessor / Successor pairs
+
+Shows the top 5 instructions that are executed before and after each instruction.
+
+## Specialization stats
+
+TBD
+
+## Specialization effectiveness
```

Comment:
All entries are execution counts. Should add up to the total number of T1 instructions executed.
Basic: Instructions that are not and cannot be specialized, e.g. `LOAD_FAST`.
Not specialized: Base instructions in a specializable family. E.g. `LOAD_ATTR`
Specialized hits: Specialized instructions, e.g. `LOAD_ATTR_MODULE` that complete
Specialized misses: Specialized instructions, e.g. `LOAD_ATTR_MODULE` that deopt

Reply:
I filed a CPython issue for the not specialized numbers problem: https://github.com/python/cpython/issues/115521